cmd/prune-junit-xml/prunexml.go

```diff
@@ -22,12 +22,15 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"regexp"
 
 	"k8s.io/kubernetes/third_party/forked/gotestsum/junitxml"
 )
 
 func main() {
 	maxTextSize := flag.Int("max-text-size", 1, "maximum size of attribute or text (in MB)")
+	pruneTests := flag.Bool("prune-tests", true,
```

Comment:
It's on by default now, as suggested in https://github.com/kubernetes/kubernetes/pull/117029#pullrequestreview-1444360343